script.js

```diff
@@ -0,0 +1,33 @@
+function calculation() {
+  let operator = document.getElementById('operator')
+  let mathOperators = operator.options[operator.selectedIndex].value
+  let answer = document.getElementById('answer')
```

Comment:
No need to search for elements from DOM each time the function runs on button click as your element ids are not changing in the html (its static)

move "document.getElementById('answer')" outside the function on top and save it in the a variable. then use the variable to get its value "<variable name>.value before calculation function is defined.

**OR like this**

let input1 = null;
let input2 = null;
function sum(){
    if (input1 == null){// set it only once if null
        input1 = documents.getElementById('num1');
    }
    if (input2 == null){// set it only once if null
        input2 = documents.getElementById('num2');
    }
    let input1Num = Number(input1.value);
    let input2Num = Number(input2.value);
    ....//do the rest of logic here;
}